client/src/components/Search.js

```diff
@@ -20,19 +20,32 @@ import { makeStyles } from "@material-ui/core/styles";
 
 import { getCityId, getRouteData, getCityName } from "../utils/skyscanner";
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
   root: {
     height: "90px",
-    width: "80%",
     background: "white",
     margin: "0 auto",
     borderRadius: "20px",
     boxShadow: "0 0 5px 5px rgba(221, 221, 240, 0.7)",
     display: "flex",
     alignItems: "center",
+    [theme.breakpoints.down("sm")]: {
+      height: "400px",
+    },
+    [theme.breakpoints.up("sm")]: {
```

Comment:
Generally you want to pick a breakpoint direction (down or up) and stick to it throughout